pkg/resource/endpoint/testdata/endpoint/cr/e_cmd_invalid_create_attempted.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: Endpoint
+metadata:
+  creationTimestamp: null
```

Comment:
metadata.name missing

Reply:
The name did not show up because when using yaml Marshal of the input file it found a typo field instead of name, and therefore ignored it.

Since the typo in the input file has been fixed, the output file has now been fixed as well.